Bottom: 23875b59a273fa19a77b8968b538b93e49c5465a
Top:    e3d25eb91fb46e103c7c2aac15e5ecc2ea4e9520
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-02-04 15:57:59 +0000

Adjust circular approximation


---

diff --git a/src/polygon.h b/src/polygon.h
index 3737f35..2b8a029 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,14 +34,14 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 /* adjustment to make the segments outline the circle rather than connect
  * points on the circle: 1 - cos (\alpha / 2) < (\alpha / 2) ^ 2 / 2
  */
-#define POLY_CIRC_RADIUS_ADJ (1.0 + M_PI / POLY_CIRC_SEGS_F * \
-                                    M_PI / POLY_CIRC_SEGS_F / 2.0)
+#define POLY_CIRC_RADIUS_ADJ 1.0 //(1.0 + M_PI / POLY_CIRC_SEGS_F * \
+                                 //   M_PI / POLY_CIRC_SEGS_F / 2.0)
 
 /* polygon diverges from modelled arc no more than MAX_ARC_DEVIATION * thick */
 #define POLY_ARC_MAX_DEVIATION 0.02
